dev/automated_tests/pubspec.yaml

```diff
@@ -15,8 +15,8 @@ dependencies:
   platform: 3.1.0
   test: 1.20.1
 
-  _fe_analyzer_shared: 34.0.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  analyzer: 3.2.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  _fe_analyzer_shared: 33.0.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  analyzer: 3.1.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
did we expect these to be downgraded?

Reply:
Oh that's really odd... Not sure why that would happen, but definitely wasn't expected.

Reply:
Good catch. I'm requesting changes while I investigate why that's getting resolved to an older version.

Reply:
SGTM. I've done a bit of digging and haven't really come up with anything.